database/factories/ProcessMaker/Models/ProcessRequestFactory.php

```diff
@@ -21,23 +21,23 @@ class ProcessRequestFactory extends Factory
      */
     public function definition()
     {
-        $process = Process::factory()->make();
+        $process = null;
```

Comment:
Since this variable is no longer used, would you consider removing it, thus simplifying the code?

```php
public function definition()
    {
        return [
            'name' => $this->faker->sentence(3),
            'data' => [],
            'status' => 'ACTIVE',
            'callable_id' => function () {
                $process = Process::factory()->create();
                $bpmnProcess = $process->getDefinitions()->getElementsByTagNameNS(BpmnDocument::BPMN_MODEL, 'process')->item(0);

                return $bpmnProcess->getAttribute('id');
            },
            'user_id' => function () {
                return User::factory()->create()->getKey();
            },
            'process_id' => function () {
                $process = Process::factory()->create();

                return $process->getKey();
            },
            'process_collaboration_id' => function () {
                return ProcessCollaboration::factory()->create()->getKey();
            },
            'process_version_id' => function (array $processRequest) {
                return Process::find($processRequest['process_id'])->getLatestVersion()->id;
            },
        ];
    }
```

Reply:
Fixed thanks!